service/src/main/kotlin/fi/espoo/evaka/placement/PlacementPlanService.kt

```diff
@@ -47,10 +47,12 @@ class PlacementPlanService(
 
         when (application.type) {
             ApplicationType.PRESCHOOL -> {
-                val term = derivePreschoolTerm(startDate)
-                val period = FiniteDateRange(startDate, term.end)
+                val preschoolTerms = tx.getActivePreschoolTermAt(startDate)
+                    ?: throw Exception("No suitable preschool term found for start date $startDate")
```

Comment:
I wonder if we should default to the next active term (if such exists) in case none is active during the requested date?

Reply:
I'll ask around to figure how end date should be defined but will merge this now anyway

Reply:
I thought this too but then decided to minimize all the guessing for now... Maybe it was a stupid decision, we'll see.